.github/workflows/release-it.yml

```diff
@@ -0,0 +1,37 @@
+# #########################################################################
+# Creates a new release using `workflow_dispatch` event trigger with `type`
+# as input to describe the type of release to create
+name: 'Release-it: Create a new release on demand'
+
+on:
+  workflow_dispatch:
+    inputs:
+      type:
+        description: 'Type. Can be `patch` `minor` or `major`'
```

Comment:
Do you need verification that the type was entered correctly?

Reply:
Nope that's handled by release-it :)